main.tf

```diff
@@ -1,31 +1,53 @@
-provider "aws" {
-  region = "eu-west-3"
-}
-
-resource "aws_vpc" "test-vpc" {
+//TODO: rename VPC
+resource "aws_vpc" "nginx_vpc" {
   cidr_block = "10.0.0.0/16"
-
+  
   tags = {
-    Name = "Premium Auto VPC"
+    Name = "Nginx VPC"
   }  
 }
```

Comment:
Naming the resource `aws_vpc.premium_auto` is okay, because in this VPC potentially won't only have Nginx instances, but other resources and containers.

Change resource name and the tag name.

Reply:
I changed resource name and tag name, and removed todo comments